packages/admin/admin/src/mui/menu/Menu.tsx

```diff
@@ -64,13 +67,14 @@ const MenuDrawer: React.FC<WithStyles<typeof styles> & MenuProps> = ({
             <Drawer
                 variant="temporary"
                 className={temporaryDrawerClasses.join(" ")}
-                open={temporaryOpen}
+                open={initialRender.current ? false : temporaryOpen}
```

Comment:
```suggestion
                // workaround for issue: https://github.com/mui/material-ui/issues/35793
                open={initialRender.current ? false : temporaryOpen}
```

Let's also put the hint here